apps/coscrad-frontend/src/components/resources/transcribed-audio/transcribed-audio-index.presenter.tsx

```diff
@@ -1,13 +1,49 @@
 import { IIndexQueryResult, ITranscribedAudioViewModel } from '@coscrad/api-interfaces';
-import { GenericIndexPresenter } from '../../../utils/generic-components/presenters/generic-index-presenter';
+import { Link } from 'react-router-dom';
 
 export const TranscribedAudioIndexPresenter = (
     indexResult: IIndexQueryResult<ITranscribedAudioViewModel>
-) => (
+) => {
     /**
      *  TODO [https://www.pivotaltracker.com/story/show/183681839]
      * We may some day read the actions and allow for bulk command execution in
      * an index view.
      */
-    <GenericIndexPresenter {...indexResult} />
-);
+    const { data: transcribedAudioItems } = indexResult;
+
+    return (
+        <div>
+            <h3>Transcribed Audio</h3>
```

Comment:
Nice stuff, Geoff! We'll refactor this to use the generic table builder before we merge in.

Reply:
done